packages/wrangler/src/deploy/index.ts

```diff
@@ -105,20 +102,14 @@ export function deployOptions(yargs: CommonYargsArgv) {
 				hidden: true,
 			})
 			.option("legacy-assets", {
```

Comment:
are we good with hiding `legacy-assets`?

Reply:
should we hide `--sites` as well?

Reply:
hmmm...good question. I'll defer to @tanushree-sharma

Reply:
for posterity, we went with:
- hide `legacy-assets`
- hide `site` & site related

in `dev`, `deploy` and `versions upload`